create-budget/azuredeploy.parameters.json

```diff
@@ -0,0 +1,57 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "budgetName": {
+      "value": "#####"
```

Comment:
add valid values here where possible - if they are subscription specific then you can use placeholders (e.g. resourceId)

Reply:
Added valid values wherever feasible and removed invalid values. Added more details about each parameter in README.md file.